ads/epeex.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+
+export function epeex(global, data) {
+
```

Comment:
You may want to use our `validateData` method here, but it is up to you.

Reply:
Hi, i dont want to use validateData

Reply:
I'm ready for merge

Reply:
ok we need to wait for travis tests to pass. I will try to keep an eye on it, but ping me if you see it before.

Reply:
@calebcordry i dont know why travis tests fail. Can you help me, i don't have touch nothing

Reply:
I have create another pull request (#15676), i can close this a use the new?